api/PerformanceObserver.json

```diff
@@ -225,7 +225,7 @@
               "version_added": "≤79"
             },
             "firefox": {
-              "version_added": null
+              "version_added": "68"
             },
             "firefox_android": {
               "version_added": null
```

Comment:
This is supported on Android too and I can't find anything that suggests it would've landed in a different version. Let's set this one too?

```suggestion
              "version_added": "68"
```

Reply:
I couldn't test on Android but yes I assume it should be supported there too.